src/components/MentorsList/MentorList.css

```diff
@@ -12,22 +8,27 @@
     transform: translateY(0);
     transition: transform 0.3s ease;
   }
-
   .mentors-wrapper.active {
     transform: translateY(190px);
     margin-bottom: 50px;
   }
 }
 
-.mentors-wrapper > .mentors-cards {
-  display: flex;
-  flex-wrap: wrap;
-  justify-content: center;
-  overflow: auto;
-  margin-top: 30px;
+@media all and (min-width: 580px) {
+  .mentors-wrapper {
+    padding-bottom: 30px;
+    margin-left: 276px;
+  }
+  .mentors-wrapper>.mentors-cards {
```

Comment:
for css performance I would change this to `.mentors-cards`. It should be the same

Reply:
As noted - changed media-widths to match Bootstrap standard.
Changed .mentors-wrapper>.mentors-cards to .mentors-cards (behavior is the same)